h2/utilities.py

```diff
@@ -160,6 +160,7 @@ def validate_headers(headers, hdr_validation_flags):
     """
     Validates a header sequence against a set of constraints from RFC 7540.
 
+    :param headers: An iterable of headers.
```

Comment:
Some naughty person wrote an incomplete argument list in the docstring, tut tut.